docs/scripts/basic.md

```diff
@@ -1,32 +1,34 @@
 ## Welcome
 
 **Facilitator**
-Welcome everyone. Thank you so much for taking the time join us together so that we can all have a more satisfying 
-democracy.
+Welcome everyone. Thank you so much for taking the time to join us together to make our democracy informed and consensual.
 
 ## Introductions
 
 **Facilitator**
-Let's begin by sharing our names and pronouns. Also, what brought you here. I'll start first.
+Let's begin by sharing our names, pronouns we'd like to be addressed by and what brought you here today. I'll start.
 
-My name is Ben. The pronouns I use are "he", "him" and "his" as well as "they", "them" and "theirs". What brings me here is the desire to see a world that works for everyone where we can all take each other seriously.
+My name is Ben. The pronouns I like to be addressed by are "he", "him" and "his" as well as "they", "them" and "theirs". What brings me here is that I want to live in a world that works for everyone where everyone is free to know what is happening and has a say in what happens.
 
 *Everyone shares their names, pronouns and purpose.*
 
 ## Process & Purpose
 
-Our main objective for the today is to come up with meaningful questions to ask the people whom we share our planet with. Some of these people are those who run for office and we are negotiating expectations with. Some are currently in office and we must help them uphold the will of the people. We shall ask questions to activists that increase our mutual understanding of each other. We may also need to help each other understand how different people come to the conclusions that they make and how to be free to make decisions that we can all be proud of.
+Our main objective for today is to invent meaningful questions to ask those we share our community with and those who represent us. We may ask those who run for office in the negotiation of expectations. We may ask those currently in office so we may help them uphold the will of the people. We may ask questions to activists that increase our mutual understanding of each other. We may also need to help each other understand how different people come to the conclusions reached. What is the least we need to do to be free to make decisions that benefit everyone in our community.
```

Comment:
"We may ask those who run for office **in the negotiation of expectations**. We may  ask (**What**) those  currently in office so we may help"  
Wording on this is a bit confusing on first sentence.

Reply:
Question mark needed on Last sentence after Community.